ts/private/ts_project_worker.js

```diff
@@ -880,6 +880,9 @@ async function emit(request) {
     debug(`# Beginning new work`);
     debug(`arguments: ${request.arguments.join(' ')}`)
 
+    const validationPath = request.arguments[request.arguments.indexOf('--bazelValidationFile') + 1]
```

Comment:
blah, too bad we still have to maintain this code :/

Reply:
Yeah :(